docs/conf.py

```diff
@@ -668,18 +674,23 @@ def _get_params(root_schema: dict, prefix: str = "", default_section: str = "")
     PACKAGE_DIR,
 ]
 
+if SYSTEM_TESTS_DIR and os.path.exists(SYSTEM_TESTS_DIR):
+    autoapi_dirs.append(SYSTEM_TESTS_DIR)
+
 # A directory that has user-defined templates to override our default templates.
 if PACKAGE_NAME == 'apache-airflow':
     autoapi_template_dir = 'autoapi_templates'
 
 # A list of patterns to ignore when finding files
 autoapi_ignore = [
     '*/airflow/_vendor/*',
-    '*/example_dags/*',
     '*/_internal*',
     '*/node_modules/*',
     '*/migrations/*',
     '*/contrib/*',
+    '**/example_sla_dag.py',
+    '**/example_taskflow_api_etl_docker_virtualenv.py',
+    '**/example_dag_decorator.py',
```

Comment:
These files had problems/not used, right?

Reply:
Pretty cryptic:

```
============================== apache-airflow ==============================
------------------------------ Error   1 --------------------
WARNING: Unknown type: placeholder

------------------------------ Error   2 --------------------
WARNING: Unknown type: placeholder
```

When I remove them. I will leave to the next poor soul that will start thinking that fixing docs building **MUST** be easy.